OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c

```diff
@@ -1843,14 +1845,26 @@ PlatformBootManagerAfterConsole (
     EfiBootManagerRefreshAllBootOption ();
   }
 
+  BOOLEAN        ShellEnabled;
+  RETURN_STATUS  RetStatus;
+
+  RetStatus = QemuFwCfgParseBool (
+                FixedPcdGetPtr (PcdEntryPointOverrideFwCfgVarName),

```

Comment:
With the new implementation approach taken it does not make much sense any more to put the fw_cfg name into a PCD.

Reply:
Fixed in v2, thanks :)